docs/ref/models/constraints.txt

```diff
@@ -282,27 +282,43 @@ PostgreSQL 15+.
 
 .. attribute:: UniqueConstraint.violation_error_code
 
-The error code used when ``ValidationError`` is raised during
-:ref:`model validation <validating-objects>`. Defaults to ``None``.
+The error code used when a ``ValidationError`` is raised during
+:ref:`model validation <validating-objects>`.
+
+If :attr:`~UniqueConstraint.fields` is set and without a
```

Comment:
If you like my other suggestion, then this would be
```
Defaults to :attr:`.BaseConstraint.violation_error_code`, when either
:attr:`.UniqueConstraint.condition` is provided or
:attr:`.UniqueConstraint.fields` is not provided.

When :attr:`.UniqueConstraint.fields` is provided without a
:attr:`.UniqueConstraint.condition`, then if :attr:`.UniqueConstraint.fields`
references multiple fields, defaults to the
:attr:`Meta.unique_together <django.db.models.Options.unique_together>` error
code. Otherwise defaults to the :attr:`.Field.unique` error code.
```